proposed_changes.txt

```diff
@@ -0,0 +1,81 @@
+Some ideas on modifying data representation.
+This commit is not intended to be merged. It is only intended for discussion.
+If it is decided that some changes make sense, then they shall be handled by multiple separate commits or pull requests.
+
+Idea 1:
+
+Use float (or in rare cases double) instead of int-based datatypes for all datapoints where fractional numbers might be relevant.
```

Comment:
I agree, I even would go further and replace all "ints/float" variants with a datatype "number". But that was discussed once, and did not get majority (it was even about "signed" vs "unsigned"

The point is, if you have number (and min/max) an implementation might still infer datatypes. in Javascript it doesn't matter anyway, in C it might